services/chat/api.go

```diff
@@ -87,6 +87,8 @@ type ChannelGroup struct {
 	Admin          bool                                     `json:"admin"`
 	Verified       bool                                     `json:"verified"`
 	Description    string                                   `json:"description"`
+	IntroMessage   string                                   `json:"introMessage"`
+	OutroMessage   string                                   `json:"outroMessage"`
```

Comment:
I'm probably missing something here, but why does this need to be part of the channel group?

Reply:
As far as I understand `ChannelGroup` is an abstraction unifying personal and community channel groups. This is how the new `getChats` API propagates information about communities. There was partial adaptation done already in the status-desktop, see: https://github.com/status-im/status-desktop/pull/5182

@richard-ramos please confirm if that was the intention and if these parameters fits here

Reply:
I guess my question was rather: why does the intro/outro message have to be part of the channel/chat ?
We already have it in the community description which we receive anyways, so can't we just grab it from there if we need it inside a channel (do we need it inside a channel anyways? I thought these were meant to be shown in some modal)

Reply:
Channel group is not channel/chat, it contains chats. In other words: `ChannelGroup == Community`.

```
type ChannelGroup struct {
	Type           ChannelGroupType                         `json:"channelGroupType"`
	Name           string                                   `json:"name"`
	Images         map[string]images.IdentityImage          `json:"images"`
	Color          string                                   `json:"color"`
	Chats          map[string]*Chat                         `json:"chats"`
	Categories     map[string]communities.CommunityCategory `json:"categories"`
	EnsName        string                                   `json:"ensName"`
	Admin          bool                                     `json:"admin"`
	Verified       bool                                     `json:"verified"`
	Description    string                                   `json:"description"`
	IntroMessage   string                                   `json:"introMessage"`
	OutroMessage   string                                   `json:"outroMessage"`
	Permissions    *protobuf.CommunityPermissions           `json:"permissions"`
	Members        map[string]*protobuf.CommunityMember     `json:"members"`
	CanManageUsers bool                                     `json:"canManageUsers"`
	Muted          bool                                     `json:"muted"`
}
```

Reply:
Ah, sorry right.
Forget my questions!